Makefile

```diff
@@ -203,6 +203,12 @@ codegen:
 
 ### Docker images ###
 
+.PHONY: ubuntu-base
+ubuntu-base:
+	@echo "===> Building antrea/base Docker image <==="
+	docker build -t antrea/base:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.base.ubuntu .
```

Comment:
should the name be `base-ubuntu` to match `antrea-ubuntu`?

Reply:
Updated.